docs/platform/advanced-targeting/custom-properties.md

```diff
@@ -144,3 +144,34 @@ You can target users in numerous ways using Custom Properties. The following lis
 **Special Users**. Sometimes organizations want to release a feature to special users only, such as users with a paid membership, those with a free trial, or those who have made large contributions to the company. In this case, some property name suggestions are `accountType` `pricingPlan` `isSubscriber` `isTrialUser` `amountContributed` etc.
 
 **User Behaviour or Preferences.** You can experiment on users based on their behaviour or preferences. Some examples include `numberOfPageVisits` `gaveConsent` `preferredColor` etc.
+
+## Custom Property Schemas
+
+Similar to [Variable Schemas](/platform/advanced-variables/variable-schemas.md), Custom Property Schemas allow you to set up a schema and define a list of possible values for a custom property, which will appear in a dropdown when setting up Targeting Rules. Labels can be defined for each value to make selections more human-readable and easier to identify. This removes the need to remember valid inputs for a specific custom property.
+
+At DevCycle, we use this feature to easily enable features for certain organizations without needing to double-check the client database or Auth0 for the correct organization ID.
```

Comment:
it's a good idea to provide a concrete example but I feel like this is too specific to how things work for us internally, especially with the mentions of the client database and Auth0

Reply:
```suggestion
At DevCycle, we use this feature to easily enable features for certain organizations without needing to double-check for the correct organization ID.
```

Reply:
@ajwootto better? or do you want me do to completely diff use case?

Reply:
something like that is better but I think it still takes a few leaps of understanding to realize that we're passing an organization ID as a custom property, and then we need to know what specific org that ID represents so we can target by their human-readable name

Reply:
```suggestion
At DevCycle, we pass each organization's ID as a Custom Property that we use to turn on Features for specific customers. Schemas allow us to set up allowable organization ID values, along with their human-readable names, which makes it easy to set up Targeting Rules without needing to double-check for a correct organization ID.
```

Reply:
@ajwootto better?